src/components/todoList/index.js

```diff
@@ -11,13 +11,10 @@ const TodoList = () => {
   const todos = useSelector(selectTodos)
   const selectIsAdding = (state) => state.todos.isAdding
   const toggleIsAdding = useSelector(selectIsAdding)
-
-  const renderedListItems = todos
-    .slice(0)
-    .reverse()
-    .map((todo) => {
-      return <TodoItem key={todo.id} todo={todo} />
-    })
+  
+  const renderedItems = todos.map((item) =>
+    <TodoItem todo={ item } />
+  )
```

Comment:
Why we done that in the separate portion? We can move that single line inside the return body, or else can make separate component.